Versions-mac.plist.in

```diff
@@ -32,7 +32,10 @@
 	</dict>
 	<key>Features</key>
 	<array>
-		<string>feature-list</string>
+		<string>sgen-concurrent-gc</string>
+		<string>activation</string>
```

Comment:
The the "Features" key doesn't exist, XS defaults to using versions. Otherwise it uses the Features list. This way the Features list can be used to say "we don't support X anymore".